corehq/motech/repeaters/management/commands/populate_caserepeater.py

```diff
@@ -68,3 +68,20 @@ def update_or_create_sql_object(self, doc):
                 "black_listed_users": doc.get("black_listed_users")
             })
         return model, created
+
+    def _get_all_couch_docs_for_model(self):
```

Comment:
Would it make sense to use `@memoized` on this? As is, if the code calls this and also `_get_couch_doc_count_for_type` appears that it will query all repeater docs twice.

Reply:
Yep. I think that would be a good idea.